controllers/namespacescope_controller.go

```diff
@@ -72,6 +72,14 @@ func (r *NamespaceScopeReconciler) Reconcile(req ctrl.Request) (ctrl.Result, err
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 
+	// enable CSVInjector for edb operator multi ns support
```

Comment:
I am wondering that would it be better to update the template directly, instead of the code?
https://github.com/IBM/ibm-common-service-operator/blob/master/controllers/constant/namespace.go#L48

@horis233 Any idea?